pages/api/products/index.ts

```diff
@@ -1,6 +1,6 @@
 import type { NextApiRequest, NextApiResponse } from 'next'
 
-import { createProduct, getProductsAPI } from '../../../controllers/products'
+import { createProduct, getProductsAPI, updateProduct } from '../../../controllers/products'
```

Comment:
Please remove this import

```suggestion
import { createProduct, getProductsAPI } from '../../../controllers/products'
```